rules_gapic/java/java_gapic.bzl

```diff
@@ -175,6 +176,17 @@ def java_gapic_library(
         "@javax_annotation_javax_annotation_api//jar",
     ]
 
+    if transport_protocol == "http":
```

Comment:
It looks like either HTTP or gRPC will be generated, not both. Would it make sense to make `transport_protocol` a boolean flag that indicates whether it's HTTP or not?

Reply:
I changed `transport_protocol` to `transport`. The name and the possible values for the argument now match the general REGAPIC spec.